### DISCUSSION TITLE
It's truly open source

### DISCUSSION THREAD

**Discussion by mc-stephen** [coollabsio/coolify#1089] [Q&A] [answered]
I have heard of open source paas services out there which are not truly open source, so i just wanted to know if this is really 100% open source or not?

Thanks.

**Comment by miguelgargallo** [comment]
Not only it's open source, also it's 100% easy to deploy.

**Comment by peaklabs-dev** [comment, marked as answer]
Yes, it is 100% open source and free and will always be.

**Comment by miguelgargallo** [comment]
Yes, trust them, its 100% on your servers, without depending on anyone, on a docker container, and fully yours, you can even take a clone of the whole server, and there will be the services, you will need to manage with ip and domains redirections, but works like a champ 100% trust
